ci-operator/step-registry/operatorhub/subscribe/cluster-logging/operatorhub-subscribe-cluster-logging-commands.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z "${CLO_SUB_INSTALL_NAMESPACE}" ]]; then
+  echo "ERROR: INSTALL_NAMESPACE is not defined"
+  exit 1
+fi
+
+if [[ -z "${CLO_SUB_PACKAGE}" ]]; then
+  echo "ERROR: PACKAGE is not defined"
+  exit 1
+fi
+
+if [[ -z "${CLO_SUB_CHANNEL}" ]]; then
+  echo "ERROR: CHANNEL is not defined"
+  exit 1
+fi
+
+if [[ "${CLO_SUB_TARGET_NAMESPACES}" == "!install" ]]; then
+  CLO_SUB_TARGET_NAMESPACES="${CLO_SUB_INSTALL_NAMESPACE}"
+fi
+
+echo "Installing ${CLO_SUB_PACKAGE} from ${CLO_SUB_CHANNEL} into ${CLO_SUB_INSTALL_NAMESPACE}, targeting ${CLO_SUB_TARGET_NAMESPACES}"
+
+# create the install namespace
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "${CLO_SUB_INSTALL_NAMESPACE}"
+  labels:
+    openshift.io/cluster-monitoring: "true"
+EOF
+
+# deploy new operator group
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: "${CLO_SUB_INSTALL_NAMESPACE}-operator-group"
```

Comment:
@IshwarKanse In our test code, the name is set to the namespace name: https://github.com/openshift/openshift-tests-private/blob/master/test/extended/logging/utils.go#L216, I'm afraid that `name: "${CLO_SUB_INSTALL_NAMESPACE}-operator-group"` might import some unexpected failures.

Same comment to ci-operator/step-registry/operatorhub/subscribe/elasticsearch-operator/operatorhub-subscribe-elasticsearch-operator-commands.sh and ci-operator/step-registry/operatorhub/subscribe/loki-operator/operatorhub-subscribe-loki-operator-commands.sh.